Makefile

```diff
@@ -152,7 +152,7 @@ go-mod-tidy:
 
 clean:
 	@echo "--> Cleaning..."
-	@rm -rf $(BUILD_DIR)/**  $(DIST_DIR)/**
+	@rm -rf $(BUILDDIR)/**  $(DIST_DIR)/**
```

Comment:
this will deleting entire /root folder on any system , it is due to `BUILD_DIR` is not set